README.md

```diff
@@ -30,21 +30,22 @@ dependencies {
 
 ## Run the Sample App
 
-The sample-app/ folder in this repository contains a very simple integration of the Forage SDK. To get it running:
+The `/sample-app/` folder in this repository contains a very simple integration of the Forage SDK. To get it running:
 
 1. [Download Android Studio](https://developer.android.com/studio).
-   i. The app was developed with Android Studio Electric Eel.
+   - The app was developed with Android Studio Giraffe and Iguana.
 2. Clone this repo to your local machine.
-3. In Android Studio, open the cloned `forage-android-sdk` project folder
-   i. Android Studio will start downloading the Gradle dependencies. Wait for dependencies to download before moving forward.
-4. [Create a bearer token](https://docs.joinforage.app/recipes/generate-a-token) with `pinpad_only` scope.
-5. Confirm your FNS number on the Forage dashboard ([sandbox](https://dashboard.sandbox.joinforage.app/login/) | [prod](https://dashboard.joinforage.app/login/)).
-6. Place your bearer token and FNS number in constants inside `sample-app/java/com.joinforage.android.example/ui/complete.flow/tokens/model/TokensUIDefaultState.kt`.
-   i. The sample-app will prompt you for a bearer token and FNS number on the first page of the app, but takes defaults from this file location.
+3. In Android Studio, open the cloned `forage-android-sdk` project folder.
+   - Android Studio will start downloading the Gradle dependencies. Wait for dependencies to download before moving forward.
+4. [Create an authentication token](https://docs.joinforage.app/reference/create-authentication-token) with `pinpad_only` scope.
+   - **Warning**: To keep your app secure, use a long-lived authentication token for client-side requests in testing and development *only*. In production, client-side requests must use [session tokens](https://docs.joinforage.app/reference/create-session-token).
```

Comment:
Can we add a step after this one for "Create a session token" like we [do here](https://arc.net/l/quote/pidrbnqu)

Reply:
We want to encourage the use of session tokens – since merchants will have to build in session expiration handling, and we want to promote the use of the more secure authentication mechanism.

Reply:
Hi @kimberleehowley We want to encourage the use of session tokens – since merchants will have to build in session expiration handling, and we want to promote the use of the more secure authentication mechanism. By omitting the step from the docs entirely, I worry this can lead to issues for the merchant.

It's your call whether we should A) remove the blurb about: "You can use auth tokens for local development..." or  B) keep the blurb and add a "Create a session token" step.

Reply:
Thanks, @djoksimo . For brevity and to focus encouragement I'm going to replace the authentication token step with the session token step that links to the docs (from the docs it's clear that you need to pass an auth token in that request). I'll keep the warning as a sub-bullet, slightly rephrased. Pushing up and re-requesting review!